app/Helpers/database/release.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use App\Models\System;
+use Illuminate\Support\Facades\Cache;
```

Comment:
It's not this line, but something in this PR is causing an error when loading a game from RAlibretro:
```
111855.649|INFO| << startsession response (500): {"Success":false,"Error":"Declaration of Laravel\\Horizon\\RedisQueue::pop($queue = null) must be compatible with Illuminate\\Queue\\RedisQueue::pop($queue = null, $index = 0)","Status":500}
```
I don't know why it isn't happening in the unit test.

Reply:
I'm having trouble reproducing this issue:

<img width="841" alt="Screenshot 2024-11-04 at 8 29 25 PM" src="https://github.com/user-attachments/assets/b1c433c2-c57f-40bb-88f7-30159775ade5">

Just for the sake of ruling things out, in latest I've reverted the _composer.json_ and _composer.lock_ changes. Does that fix things for you on latest?

Reply:
I can confirm that the problem does not occur on 9f375249f8872adc641e369ff0941091387edc5c but does on de4bc5f8946807a0583624ddf3c93de2724b1fff.

And I can reproduce from the command line:
```
$ curl "localhost:64000/dorequest.php?r=startsession&u=Jamiras&t=redacted&g=1&m=1bc674be034e43c96b86487ac69d9293"
{"Success":false,"Error":"Declaration of Laravel\\Horizon\\RedisQueue::pop($queue = null) must be compatible with Illuminate\\Queue\\RedisQueue::pop($queue = null, $index = 0)","Status":500}
```

The problem also doesn't occur if I comment out this line: https://github.com/RetroAchievements/RAWeb/blob/e735815e6e112bf9426b5ad85c59b16204ce0391/public/dorequest.php#L541